src/encoded/static/components/browse/components/above-table-controls/SelectedFilesDownloadButton.js

```diff
@@ -151,26 +151,36 @@ class SelectedFilesDownloadModal extends React.PureComponent {
     }
 
     handleAcceptDisclaimer(){
+        const { context } = this.props;
+        const { filters = null, display_title, title } = context || {};
+        const eventObj = {
+            name: display_title || title
+        };
+
+        if (filters) {
+            eventObj.currentFilters = analytics.getStringifiedCurrentFilters(filters);
+        }
+
+        analytics.event("SelectedFilesDownloadModal", "Accepted Disclaimer", eventObj);
         this.setState({ 'disclaimerAccepted' : true });
     }
 
     render(){
-        const { onHide, filenamePrefix, selectedFiles } = this.props;
+        const { onHide, filenamePrefix, selectedFiles, fileCountUnique, fileCountWithDupes } = this.props;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['fileCountWithDupes' is assigned a value but never used.](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=4660599)